config/ftbquests/quests/chapters/bigger_reactors.snbt

```diff
@@ -0,0 +1,1071 @@
+				""
+				"The &cPower Tap&r extracts power, and is required for us to get power from the turbine."
+				""
+				"You can also make a &eComputer Port&r, but it is completely optional. It allows you to interact with the Turbine with ComputerCraft using scripts."
+			]
+			dependencies: ["4415C9F8DA2D7E68"]
+			id: "186731580B14F9D2"
+			tasks: [
+				{
+					id: "5E9CEEEE330B1DE0"
+					type: "item"
+					item: "biggerreactors:turbine_fluid_port"
+				}
+				{
+					id: "18EB6570007F534A"
+					type: "item"
+					item: "biggerreactors:turbine_power_tap"
+				}
+			]
+			rewards: [
+				{
+					id: "6F76A59F202AD944"
+					type: "item"
+					item: "biggerreactors:turbine_casing"
+					count: 4
+					random_bonus: 4
+				}
+				{
+					id: "7BC028EFCCFAF39A"
+					type: "xp"
+					xp: 10
+				}
+			]
+		}
+		{
+			title: "Creating The Turbine Shaft"
+			x: -4.0d
+			y: 10.0d
+			shape: "diamond"
+			description: [
+				"To get a Turbine spinning, we'll need these &crequired&r components:"
+				""
+				"- &9Rotor Bearings&r are placed at each end of the shaft of the turbine. While the shaft technically does not have to be centered in the turbine, it's generally the optimal place for it. You will need exactly 2 per Turbine, one at the top and one at the base."
+				""
+				"- &eRotor Shafts&r extend from one Rotor Bearing to the other, creating the shaft for the Turbine. The shaft can be vertical or horizontal."
+				""
+				"- &9Rotor Blades&r make the rotor spin. These are placed onto the Rotor Shafts and can be multiple blocks in length."
+				""
+				"Here is an example of a Horizontal Shaft setup for a Turbine, with the Turbine Coil of Gold on the right side."
+				""
+				"{image:atm:textures/questpics/biggerreactors/turbine_horizontal_shaft.png width:200 height:125 align:1}"
+			]
+			dependencies: ["4415C9F8DA2D7E68"]
+			min_width: 300
+			id: "67AFCBCE7AAC3089"
+			tasks: [
+				{
+					id: "69497C419271A8F2"
+					type: "item"
+					item: "biggerreactors:turbine_rotor_bearing"
+					count: 2L
+				}
+				{
+					id: "05559BFC34BEBF4A"
+					type: "item"
+					item: "biggerreactors:turbine_rotor_shaft"
+					count: 2L
+				}
+				{
+					id: "42F16075D25E4A94"
+					type: "item"
+					item: "biggerreactors:turbine_rotor_blade"
+					count: 4L
+				}
+			]
+			rewards: [
+				{
+					id: "75FCB874DBC8BC86"
+					type: "item"
+					item: "biggerreactors:turbine_rotor_blade"
+					count: 4
+				}
+				{
+					id: "6A0A9AAD57654338"
+					type: "item"
+					item: "biggerreactors:turbine_glass"
+					count: 4
+					random_bonus: 4
+				}
+				{
+					id: "433D5587FE499E95"
+					type: "xp"
+					xp: 10
+				}
+			]
+		}
+		{
+			title: "Turbine Coils"
+			x: -3.0d
+			y: 11.0d
+			shape: "diamond"
+			description: [
+				"&dTurbine Coils&r are blocks placed around a shaft at one end of a Turbine. These are required to generate power from the turbine."
```

Comment:
I forgot to note this the first time around, I've seen people confused about the requirements for a coil and what you can or cant do with them, partially because the mechanics have changed.

1. Coils do not need to be placed in full rings, single blocks can be used.
2. Coils placed in a closer (square) ring to the rotor shaft will have a higher drag, this can be used to better fine tune a turbine or to get more out of the same size turbine than previously possible. this is the new mechanic, its mostly intended to help with *massive* turbines, where doing this also gets very expensive very fast
3. Coils do not need to be a single material, you can mix and match